### PR TITLE
update use cases

### PR DESCRIPTION
**Pull request by markus2330**

### CODE REVIEW

**Comment by horenso** [comment]
Thank you for listen the wanted commands in more detail, that's very helpful!

File: doc/usecases/assigned/nextcloud_talk.md [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
+  - The user writes `indoor climate?` (or `Innenklima?`) and the indoor environment data gets reported.
+  - The user writes `status?` (or `Status?`) and opensesame reports the user's nextcloud status into the chat.
+  - The user writes `sensors?` (or `Sensoren?`) and the current sensor data gets reported.
+  - The user writes `alarm!` (or `Alarm!`) and a bell alarm gets triggered or canceled.
+  - The user writes `bell!` (or `Glocke!`) and a bell is ringed.
+  - The user writes `quit!` (or `beenden!`) and opensesame quits (and automatically restarts).
+  - The user writes `time?` (or `Uhrzeit?`) and opensesame report their current time.
+  - The user writes `play [audio file]!` (or `abspielen [audio file]!`) and an audio file is played (default bell).
+  - The user writes `code?` and gets a list of names and codes.
+  - The user writes `code add <name> <pin>!`, `code del <name>!` or `code set <name> <pin>!` (or `pin hinzufügen/löschen/ändern!`) in the chat and new validator codes are added or removed.
```
[the remark this answers is not in the document]
Comment (horenso) [reply doc/usecases/assigned/nextcloud_talk.md:32]: does that mean `code hinzufügen`, `pin add`, `code add` and `pin hinzufügen` all work? ^^

[the remark this answers is not in the document]
Comment (markus2330) [reply doc/usecases/assigned/nextcloud_talk.md:32]: and also `c h` or similar unambiguous shortcuts.

File: doc/usecases/assigned/nextcloud_talk.md [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
+- **Alternate Scenario:** 
+  - The user pre- or postfixes one of the commands with `@user` and only the specific opensesame instance with `user` as Nextcloud user responds to the question.
+    Example: `@garage öffnen!`
```
[the remark this answers is not in the document]
Comment (markus2330) [reply doc/usecases/assigned/nextcloud_talk.md]: Ok, in 6263d5a7a38d64ad4a842cebc88c8da7ab904d8f I removed the postfixing and clarified which config is meant (it was missing in the spec file).

**Comment by markus2330** [comment]
@horenso please approve if everything is clear or ask further questions :rocket:

**Review by horenso — APPROVED** [review]

**Pull request merged**